### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by omahs**
Fix typos

### CODE REVIEW

File: internal/common/README.md [hunk 1]
```diff
@@ -36,4 +36,4 @@ To enforce an update on those files, you can delete them and run the code again/
 
 ### Usage:
 
-The package is designed to be used by calling the `SetMockAVSs()` function, which sets up the data structures with the latest versions of the repositories. The data is then written to the `mock-avs-versions.yml` file. If the data is older than an hour, it fetches the latest data from GitHub and updates the file.
\ No newline at end of file
+The package is designed to be used by calling the `SetMockAVSs()` function, which sets up the data structures with the latest versions of the repositories. The data is then written to the `mock-avs-versions.yml` file. If the data is older than an hour, it fetches the latest data from GitHub and updates the file.
```
Comment (shrimalmadhur) [inline internal/common/README.md]: I don't see any difference here. not sure why github added this. can you revert somehow ? @omahs

Comment (omahs) [reply internal/common/README.md]: Hey sure, just fixed this

**Comment by shrimalmadhur** [comment]
could you please also resolve the commit @omahs . Appreciate it.

**Comment by omahs** [comment]
Hey @shrimalmadhur I just squashed the commit

**Comment by shrimalmadhur** [comment]
> Hey @shrimalmadhur I just squashed the commit

oh I meant to say resolve the conflict. my bad. there's a conflict in Readme

**Pull request closed without merging**